data/lib/core/player.lua

```diff
@@ -545,15 +545,29 @@ local function getStaminaBonus(staminaMinutes)
 	end
 end
 
+local function getLowLevelBonus(level)
+	if level > 1 and level <= 50 then
+		return 50
+	else
+		return 0
+	end
+end
```

Comment:
This is not the correct value though :thinking: 

In connection with data/events/scripts/player.lua it should be:
```lua
local bonus = expBonus.bonus * math.min(1, math.max(0, (expBonus.maxlevel - level) / (expBonus.maxlevel - expBonus.minlevel))) -- the level bonus as a ratio
local bonusPercent = math.ceil(multiplier * 100) -- displayed in the client
local multiplier = 1 + bonus -- for multiplying with the raw XP
```


Not sure how to make this all work together though

Reply:
the code seems to me to work as it should now (the player is level 8 and receiving 87% experience bonus):
![image](https://github.com/otland/forgottenserver/assets/26801045/e34b483b-1642-4b7c-b36b-a66653064788)

Reply:
Awesome implementation!

Reply:
> Awesome implementation!

thank you <3, if you accepted it would be great 😂